common/mqtt_client/tests/packet_size_tests.rs

```diff
@@ -81,19 +79,13 @@ async fn subscribe_errors(pub_client: &Client) -> Result<(), MqttClientError> {
     Ok(())
 }
 
-async fn start_broker_local(cfile: &str) -> anyhow::Result<()> {
-    let config: Config = confy::load_path(cfile)?;
-    let (mut router, _console, servers, _builder) = async_locallink::construct_broker(config);
-    let router = tokio::task::spawn_blocking(move || -> anyhow::Result<()> { Ok(router.start()?) });
-    servers.await;
-    let _ = router.await;
-    Ok(())
-}
-
 async fn subscribe_until_3_messages_received() -> Result<(), anyhow::Error> {
     let sub_filter = TopicFilter::new("test/hello")?;
-    let client =
-        Client::connect("subscribe", &mqtt_client::Config::default().with_port(5883)).await?;
+    let client = Client::connect(
+        "subscribe",
+        &mqtt_client::Config::default().with_port(58584),
```

Comment:
As you are using your ports in multiple places I suggest you make them `const` and give them names such that when used for server and client you have to change it only in one place.

Reply:
done